### PR TITLE
Fix Terms and Privacy links in footers

### PR DESCRIPTION
**Pull request by jrubenoff**

### CODE REVIEW

**Comment by ajb** [comment]
:question:  :grey_question: 

I already did this in https://github.com/dobtco/dvl-core/commit/7039a9b781f90b9f3cb72e4e10ca206c5b5dc67c.

File: lib/dvl/components/footer.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def content
 
         ul {
           li { a t('dvl_core.footer.status'), href: 'http://status.dobt.co', target: '_blank' }
-          li { a t('dvl_core.footer.legal'), href: 'http://www.dobt.co/terms/', target: '_blank' }
+          li { a t('dvl_core.footer.terms'), href: 'http://dobt.co/terms/', target: '_blank' }
```
Comment (ajb) [inline lib/dvl/components/footer.rb]: the correct domain is `www.dobt.co`

File: spec/dummy/app/views/home/splash.rb [hunk 2]
```diff
@@ -37,7 +37,7 @@ def main
                   a 'Terms of Service', href: 'http://www.dobt.co/terms/'
                 }
                 li {
-                  a 'Privacy Policy', href: 'http://www.dobt.co/privacy/'
+                  a 'Privacy Policy', href: 'https://www.dobt.co/privacy/'
```
Comment (ajb) [inline spec/dummy/app/views/home/splash.rb]: `https` is incorrect.

**Comment by jrubenoff** [comment]
Ah, sorry, my local version is just outdated.

**Pull request closed without merging**